ui/src/pages/DirectoryTraversal.js

```diff
@@ -1,5 +1,5 @@
 import React,{useState, useEffect} from 'react';
-import '../style/ChallengeSet1.css';
+// import '../style/ChallengeSet1.css';
```

Comment:
remove commented out code if not going to reuse.

Reply:
Directory Traversal challenge was left out of the Challenge-pages group folder when I was working on miragting from read local to read API and delete the locals. I left it there in case we wanted to re-use it

Reply:
This got removed